src/CMakeLists.txt

```diff
@@ -105,12 +105,12 @@ add_subdirectory(common)
 if (BUILD_AGENTS)
     add_subdirectory(agents)
 endif()
-if (BUILD_MODULES)
-    add_subdirectory(modules)
-endif()
 if (BUILD_PLATFORM)
     add_subdirectory(platform)
 endif()
+if (BUILD_MODULES)

```

Comment:
Why this change? Please revert

Reply:
the order must be changes as i rely on the newly exposed `osconfig-platform-lib` which is available in the platform.

Reply:
we are leaving the order changed, as per our offline conversation